src/renderer/components/ft-share-button/ft-share-button.js

```diff
@@ -14,6 +14,10 @@ export default Vue.extend({
     id: {
       type: String,
       required: true
+    },
+    timestamp: {
```

Comment:
This probably won't be the best way to retrieve the timestamp in the scenario.  With how you're trying to set it up, you may have to instead call your `getTimestamp` function from the child element here.  You can do this with the following syntax `this.$parent.getTimestamp()`.

Reply:
I wasn't aware that was an option, and yes, that would be a big improvement. I'll look into doing it that way and see if I run into any issues.